services/helloworld/src/Site.hs

```diff
@@ -114,18 +120,37 @@ tokenOk :: Maybe ByteString -> Handler App App Bool
 tokenOk Nothing = return False
 tokenOk (Just token) = do
     hwConfig <- gets aHWConfig
-    let sid = serviceId hwConfig
-        key = serviceKey hwConfig
-        url = thentosBackendUrl hwConfig <> "/session/" <> urlEncode token <> "/login/" <> urlEncode sid
+    let sid = encodeUtf8 $ serviceId hwConfig
+        url = thentosBackendUrl hwConfig <> "/session"
+        json_tok = "{\"fromSessionToken\":\"" <> token <> "\"}"
```

Comment:
I would prefer:

    Aeson.object ["fromSessionToken" .= token]

and then in the next line:

    reqBody = RequestBodyBS . cs . Aeson.encode $ json_tok